src/feedback/feedback.request.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsNotEmpty, IsString, IsUUID } from 'class-validator';
+
+export class CreateFeedbackRequestDto {
+    @IsUUID()
+    @IsNotEmpty()
+    userId!: string;
```

Comment:
User A will be able to post feedback for User B because he can specify userId of a different user.
userId should be infered from the user object and should not be passed as a DTO param